mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -357,7 +357,9 @@ public void refreshUI() {
 
     @Override
     public void onClick(View v) {
-        if (controlCenterSettingsData == null) {
+
+        if (controlCenterSettingsData == null ||
+                GeckoBundleUtils.safeGetBundle(controlCenterSettingsData, "data") == null) {
```

Comment:
as per Ravjit comment and implementation of safeGetBundle it won't return null, it creates empty bundle and fill it.
so you can check if the bunlde empty, and doesn't contain any keys using:
GeckoBundleUtils.safeGetBundle(controlCenterSettingsData, "data").isEmpty()